### PR TITLE
PL: teacher application form improvements

### PR DESCRIPTION
**Pull request by breville**
# Before

## section 1
teacher application original page 1

## section 2
teacher application original page 2

## section 3
teacher application original page 3

## thanks
screenshot 2019-02-12 09 29 40

# After

## section 1

teacher application new page 1

## section 1 - nominated

teacher application new page 1 nominated

## section 2
teacher application new page 2

## section 3
teacher application new page 3

## thanks
screenshot 2019-02-12 10 03 23

### CODE REVIEW

File: apps/src/code-studio/pd/application/teacher1920/Section1AboutYou.jsx [hunk 1]
```diff
@@ -89,25 +94,83 @@ export default class Section1AboutYou extends LabeledFormComponent {
...
+        <h3>What’s in this application and how long will it take?</h3>
+        <p>
+          This application should take 10 - 15 minutes to complete. Fields
+          marked with a <span style={{color: 'red'}}>*</span> are required. Here
+          are the sections you will be asked to fill out:
+        </p>
+        <ul>
+          <li>
+            <span style={styles.bold}>Section 1: About you</span>
+            &nbsp; (Your and your principal’s contact info)
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application/teacher1920/Section1AboutYou.jsx]: It looks like the principal's contact info is now in section 2

**Review by clareconstantine — APPROVED** [review]
One small thing, looks good otherwise!

File: apps/src/code-studio/pd/application/teacher1920/Section1AboutYou.jsx (hunk 1 above)
Comment (breville) [reply apps/src/code-studio/pd/application/teacher1920/Section1AboutYou.jsx]: Great catch.  I missed this change in the spec.

File: apps/src/code-studio/pd/application/teacher1920/Teacher1920Application.jsx [hunk 2]
```diff
@@ -26,8 +26,8 @@ export default class Teacher1920Application extends FormController {
   getPageComponents() {
     return [
       Section1AboutYou,
-      Section2ChooseYourProgram,
       Section3TeachingBackground,
+      Section2ChooseYourProgram,
```
Comment (breville) [inline apps/src/code-studio/pd/application/teacher1920/Teacher1920Application.jsx:30]: Note that I chose to not rename the classes & files for these, despite switching their order, just to reduce churn on the source code.

**Pull request merged**